src/ansys/fluent/core/fluent_connection.py

```diff
@@ -1,6 +1,10 @@
 from ctypes import c_int, sizeof
+from dataclasses import dataclass
```

Comment:
@prmukherj n.b. you are also looking at updating this module currently. I don't know if you already started. If so, there will be conflicts.